CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ set(GR_SWIG_MIN_VERSION "3.0.8")
 set(GR_CMAKE_MIN_VERSION "3.5.1")
 set(GR_MAKO_MIN_VERSION "0.4.2")
 set(GR_PYTHON_MIN_VERSION "3.6.5")
+set(GR_NUMPY_MIN_VERSION "1.13.0")
```

Comment:
Note: This is what 18.04 has.

Reply:
Seems like for master this is a good floor